test/iam.ts

```diff
@@ -110,20 +113,14 @@ describe('IAM', () => {
   });
 
   describe('setPolicy', () => {
-    const policy = {etag: 'ACAB'};
-
-    it('should throw an error if a policy is not supplied', () => {
```

Comment:
We should probably not delete this test

Reply:
in setPolicy function, policy is a required parameter with type Policy
https://github.com/googleapis/nodejs-pubsub/blob/master/src/iam.ts#L234
and after declaring explicit type it will give error, as it will not allow you to do what we are trying to do in this test case due to explicit type declaration (we my use iam as any type than this test may look relevant).

Reply:
fix the changes as suggested on another PR
https://github.com/googleapis/nodejs-pubsub/pull/502